.travis.yml

```diff
@@ -33,6 +33,8 @@ script:
   - npm run lint-ci
   - npm run build-dll
   - |
-    if [ "$TRAVIS_OS_NAME" == "osx" ]; then 
-      npm run package-all;
+    if [ $TRAVIS_OS_NAME == "osx" ]; then 
+      rm -f release/auth.md
```

Comment:
why you remove something that doesn't exist?

Reply:
right.